mmhuman3d/core/conventions/keypoints_mapping/gta.py

```diff
@@ -0,0 +1,102 @@
+GTA_KEYPOINTS = [  # idx - original name
+    'gta_head_top',  # 00  - head_top
+    'head',  # 01  - head_center
+    'neck',  # 02  - neck
+    'gta_right_clavicle',  # 03
+    'right_shoulder',  # 04  - right_shoulder
+    'right_elbow',  # 05  - right_elbow
+    'right_wrist',  # 06  - right_wrist
+    'gta_left_clavicle',  # 07
+    'left_shoulder',  # 08  - left_shoulder
+    'left_elbow',  # 09  - left_elbow
+    'left_wrist',  # 10  - left_wrist
+    'spine_2',  # 11  - spine0
+    'gta_spine1',  # 12
+    'spine_1',  # 13  - spine2
+    'pelvis',  # 14  - pelvis
+    'gta_spine4',  # 15
+    'right_hip',  # 16  - right_hip
+    'right_knee',  # 17  - right_knee
+    'right_ankle',  # 18  - right_ankle
+    'left_hip',  # 19  - left_hip
+    'left_knee',  # 20  - left_knee
+    'left_ankle',  # 21  - left_ankle
+    'gta_SKEL_ROOT',  # 22
+    'gta_FB_R_Brow_Out_000',  # 23
```

Comment:
What is `IK`, `RB`, `MH`, `FB`, `SKEL`, `FACIAL`? 

For consistency, you might want to (1) keep everything to lowercase and (2) reduce using acronyms i.e. rename `FB_R_Brow_Out_000` to something more comprehensible

Reply:
These are original keypoint names of GTA game engine and we do not know the actual meanings of some of them. As discussed offline, we may need to refactor the pipeline. For now, these keypoints are not used.